stories/Avatar/AvatarLinkWithTooltip.js

```diff
@@ -8,6 +8,13 @@ import {
 
 const AvatarLinkWithTooltip = () => {
   const ids = ['Example1', 'Example2', 'Example3', 'Example4']
+
+  ids.map((id, i) => {
```

Comment:
Not sure of this strategy.
These ids are associated with several AvatarIcon components in this file: perhaps refs are missing? Can't we use refs instead?